sheu@chromium.org: trimmed from 3.4 kernel patch

From e5b2998def807693aa5796112423117022318db2 Mon Sep 17 00:00:00 2001
From: John Sheu <sheu@chromium.org>
Date: Fri, 4 Jan 2013 19:08:53 -0800
Subject: [PATCH] CHROMIUM: v4l2/exynos: move CID enums into videodev2.h

Move some #defines for V4L_CID_* values out of gsc-core.h and into
videodev2.h, where they belong, as they are part of the userspace API.

Signed-off-by: John Sheu <sheu@google.com>

BUG=chromium-os:37294
BUG=chrome-os-partner:10057
TEST=local build, run on snow

Change-Id: Ib06cd97f8c294a0d5f42f0b2adfefe4d761b256f
---
 drivers/media/video/exynos/gsc/gsc-core.h |   12 ------------
 include/linux/videodev2.h                 |   13 ++++++++++++-
 2 files changed, 12 insertions(+), 13 deletions(-)

diff --git a/include/linux/videodev2.h b/include/linux/videodev2.h
index 09f7f6d..405313f 100644
--- a/include/linux/videodev2.h
+++ b/include/linux/videodev2.h
@@ -1342,6 +1342,18 @@ enum v4l2_colorfx {
 /* last CID + 1 */
 #define V4L2_CID_LASTP1                         (V4L2_CID_BASE+42)
 
+#define V4L2_CID_CACHEABLE                      (V4L2_CID_LASTP1 + 1)
+#define V4L2_CID_TV_LAYER_BLEND_ENABLE          (V4L2_CID_LASTP1 + 2)
+#define V4L2_CID_TV_LAYER_BLEND_ALPHA           (V4L2_CID_LASTP1 + 3)
+#define V4L2_CID_TV_PIXEL_BLEND_ENABLE          (V4L2_CID_LASTP1 + 4)
+#define V4L2_CID_TV_CHROMA_ENABLE               (V4L2_CID_LASTP1 + 5)
+#define V4L2_CID_TV_CHROMA_VALUE                (V4L2_CID_LASTP1 + 6)
+/* for color space conversion equation selection */
+#define V4L2_CID_CSC_EQ_MODE                    (V4L2_CID_LASTP1 + 8)
+#define V4L2_CID_CSC_EQ                         (V4L2_CID_LASTP1 + 9)
+#define V4L2_CID_CSC_RANGE                      (V4L2_CID_LASTP1 + 10)
+#define V4L2_CID_GLOBAL_ALPHA                   (V4L2_CID_LASTP1 + 11)
+#define V4L2_CID_CODEC_DISPLAY_STATUS           (V4L2_CID_LASTP1 + 12)
 
 /*  MPEG-class control IDs defined by V4L2 */
 #define V4L2_CID_MPEG_BASE 			(V4L2_CTRL_CLASS_MPEG | 0x900)
-- 
1.7.8.6

